### ISSUE TITLE
[DELIVERED_CHANGES_REQUIRED] AUDIT-A|PR1043|LEARNED20M-LAUNCH-INPUT-V2|EXACTHEAD-CFCD3CCB|V1

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1155]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
INSTANCE_TAG: SOL-AUD1043-8R2M
SWARM_LANE_KEY: AUDIT-A|PR1043|LEARNED20M-LAUNCH-INPUT-V2|EXACTHEAD-CFCD3CCB|V1
PREFERRED_WORK_KIND: INDEPENDENT_VERIFY / D10 / TRAINING_READINESS
PARENT_ISSUES: #11 #13 #548 #723 #802 #828 #835 #990 #1040 #1043
STATUS: ACTIVE_PENDING_POSTCLAIM_ARBITRATION
PRODUCT_PR: #1043
TARGET_HEAD: cfcd3ccb999a8e22a5c999c36428778debf5720a
TARGET_BASE: c8914601deabd95365c14c90e94482e84f37fe94
EXACT_HEAD_SHARED_CI: 34533478648=SUCCESS
PRODUCT_OWNER: #1040
INTEGRATOR: #1045

PRECLAIM ARBITRATION:
- exact OPEN search for `PR1043 audit launch input two-clean V2` returned no issue;
- exact-head search for `cfcd3ccb999a8e22a5c999c36428778debf5720a audit` returned only permanent AUDIT-A #13, Product #1040 and integrator #1045, with no independent exact-head verifier;
- losing duplicate PR #1014 remains evidence only; this audit targets canonical #1040 Product PR #1043 and does not revive #1014.

OWNED SURFACE:
- read-only exact-head audit of PR #1043 only;
- verify canonical V2 two-clean proof is independently identity-bound and closed-world before launch-input composition;
- verify exact proof/materialization/unique-loss continuity and positive non-manufactured unique causal-loss supply;
- verify replay/padding/cross-document/source-byte relabelling cannot increase unique supply or requested exposure;
- verify exact carrier Git SHA / ModelSpec / InitSpec and exact terminal carrier workflow head/run/evidence are independently bound;
- verify bool/int aliases, malformed hashes, stale/red/nonterminal CI, candidate-authored expected identities, mixed-lineage authorities and self-consistent substitution fail closed;
- verify emitted launch authority is deterministic, closed-world, self-hashed, text-free, authorizes neither training nor compute, keeps authorized optimized-target exposure exactly zero, and does not consume final-test outcomes;
- bind verdict only to this exact head and exact terminal shared CI.

AVOID:
- no Product mutation on #1043/#1040;
- no merge; #1045 owns integration while current;
- no corpus payload execution, tokenizer fit, optimizer/model training, checkpoint learning or final-test access;
- no paid compute, external LLM/API data/intelligence, or foreign pretrained weights.

ACCEPTANCE:
- postclaim arbitration still finds no earlier valid independent auditor;
- exact Product head/base unchanged and shared CI remains terminal SUCCESS;
- no self-consistent candidate object can teach the verifier its own expected proof/ledger/carrier/workflow identities;
- terminal V2 proof, materialization and unique-loss ledger form one exact lineage with strictly positive integral non-bool unique supply;
- requested exposure is positive integral non-bool and <= unique supply, with replay/padding/replacement unable to manufacture capacity;
- exact carrier SHA/ModelSpec/InitSpec and workflow head/run/evidence are independently fixed and terminal-success;
- output truth boundary cannot be widened after resealing;
- verdict is PASS_EXACT_HEAD/PASS_FOR_INTEGRATION_MECHANICS_ZERO_CREDIT, CHANGES_REQUIRED, or BLOCKED with exact reproduction.

TRUTH_BOUNDARY:
REAL_DATA_EXECUTED=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
TERMINAL AUDIT VERDICT — `CHANGES_REQUIRED` on PR #1043 exact head `cfcd3ccb999a8e22a5c999c36428778debf5720a`, exact-head CI `34533478648=SUCCESS`.

Delivered `AUD1043-001 HIGH`: local launch-input `_verify_ledger()` rejects only exact duplicate `(document_id,start,end)` tuples, while canonical V2 unique-loss semantics reject replay of every logical `(document_id,target)` position. A resealed ledger containing partially overlapping ranges such as `[1,8)` and `[4,8)` can double-count 4 logical targets, inflate `one_pass_unique_nonignored_causal_loss_positions`, and be emitted as `READY_FOR_READINESS_BINDING` if its exact resealed identity is supplied. Existing replay test duplicates the entire range and does not cover partial overlap.

Full finding/minimal repair: PR #1043 comment `5628821779`.

No Product mutation, no merge, no corpus/training/final-test execution. Product remains #1040; integration remains #1045 while current. Audit lease released.

REAL_DATA_EXECUTED=false; TRAINING_EXECUTED=false; LEARNED_WEIGHTS_CREATED=false; AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0; FINAL_TEST_ACCESSED=false; PAID_COMPUTE_USED=false; FOREIGN_PRETRAINED_WEIGHTS=false.
